googletest/docs/Primer.md

```diff
@@ -23,6 +23,40 @@ So let's go!
 _Note:_ We sometimes refer to Google C++ Testing Framework informally
 as _Google Test_.
 
+# Beware of the nomenclature #
+
+_Note:_ There might be some confusion of idea due to different
+definitions of the terms _Test_, _Test Case_ and _Test Suite_, so beware
+of misunderstanding these.
+
+Historically, the Google C++ Testing Framework started to use the term
+_Test Case_ for grouping related tests, i.e. in the same sense as
+current publications, including the International Software Testing
+Qualifications Board ([ISTQB](http://www.istqb.org/)) and different
+textbooks on Software Quality, are using the term _[Test
+Suite](http://glossary.istqb.org/search/test%20suite)_.
```

Comment:
This might be easier for humans to parse if written as "Historically, the Google C++ Testing Framework started to use the term Test Case for grouping related tests, whereas current publications including the International Software Testing Qualifications Board (ISTQB) and various textbooks on Software Quality use the term Test Suite for this."

That way the reader is alerted earlier that this is a distinction, not a similarity.